### DISCUSSION TITLE
Plot claims

### DISCUSSION THREAD

**Discussion by Misquoth** [TownyAdvanced/Towny#5659] [Q&A] [answered]
Is there any method for claiming plots with a selection tool as opposed to player location? Most of my players find the claim process to be cumbersome. I'm looking at moving from GriefPrevention, but a large contingent of my players have brought this up as an issue.

Thanks

**Comment by LlmDl** [comment, marked as answer]
Towny is something used by players and generally speaking you cannot trust players with cuboid tools. Towny's claiming process is a system of cells on a grid. There's no other way to claim land.

**Reply by Misquoth** [reply]
Bummer. Even a tool that selected the current plot would be great as opposed to a command or a server command that I could put into my own script. I hope a path for an alternative method is considered in the future.

Thanks
